qcodes/instrument_drivers/oxford/kelvinox.py

```diff
@@ -0,0 +1,534 @@
+# OxfordInstruments_Kelvinox_IGH.py class, to perform the communication between the Wrapper and the device
```

Comment:
OxfordInstruments_Kelvinox_IGH.py to either OxfordInstruments_Kelvinox_IGH or kelvinox.py